js/components/controlBar.js

```diff
@@ -232,6 +233,13 @@ var ControlBar = React.createClass({
     }
   },
 
+  handleMultiAudioClick: function () {
+    this.configureMenuAutofocus(CONSTANTS.MENU_OPTIONS.MULTI_AUDIO);
+
+    this.togglePopover(CONSTANTS.MENU_OPTIONS.MULTI_AUDIO);
+    this.closePopovers();
```

Comment:
I think we should probably update `this. closePopovers()` in order to include the newly added one. When calling `this. closePopovers()`  I would expect it would close all (multi audio included). We could possibly add a parameter to that function in order allow closing all except the one that is specified, that way we can avoid manually closing each other popover when a new one is opened.

Reply:
@jpfranco yes, I wanted to do it, but now we do not have approved UI, and maybe the design will be changed (maybe there will be no popovers or be only one).

Reply:
@KseniiaSokolovaEpam I think there's a good chance that we'll forget to add this later if the UI does end up using a popover. It might be better to include this earlier, but I leave it up to you